shared/configuration/shared/configuration/src/airflow_shared/configuration/config_templates/unit_tests.cfg

```diff
@@ -0,0 +1,139 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We shouldn't need to move this

Reply:
Yeah had moved that initially, but isnt relevant anymore. Will undo

Reply:
Removed in ceeaf9a5c2